source/app/service-providers/commands/dir-remove-project.ts

```diff
@@ -22,11 +22,12 @@ export default class DirRemoveProject extends ZettlrCommand {
 
   /**
     * Remove the project of a directory
-    * @param {String} evt The event name
-    * @param  {Object} arg The hash of a directory.
+    *
+    * @param  {string} evt  The event name
+    * @param  {any}    arg  The payload
     */
-  async run (evt: string, arg: any): Promise<void> {
-    let dir = this._app.workspaces.findDir(arg.path)
```

Comment:
The commands have been entirely moved from calling `this._app.workspaces` to `this._app.fsal`, which implements the same functionality, but on the bare metal file system syscalls instead of the tree structure.